docker/README.md

```diff
@@ -0,0 +1,52 @@
+# gpusimilarity Docker Container
+
+The preferred method of using Docker is to just to download the publicly
+released pre-built docker images.  In addition, the enclosed Dockerfile can be
+built to produce an image capable of running GPUSimilarity.
+
+## Requirements
+* nvidia-docker (regular docker will not expose GPUs)
+* Newest versions of nvidia drivers
+* Recommended for production:  systemd / nginx
+
+## How to use pre-built container
+Both releases and master are always built and uploaded publicly already.  You
+can access them via:
+For master:
+```
+docker pull klorton/gpusimilarity:latest
+```
+
+For releases:
+
+```
+docker pull klorton/gpusimilarity:v1.0
+```
+
+The underlying requirements will change very little, so after the initial
+build/pull new docker layers will only require a few megabytes for
+gpusimilarity changes.
+
+## How to build (recommended only for development)
+From the docker directory:
+```
+docker build -t gpusim:internal .
+```
+
+## How to use to build a fingerprint file
+TODO
+
+## How to start a gpusimilarity server interactively
```

Comment:
I think it's worth adding a line describing how to test that the server is actually working once it starts running.

Reply:
Yep good point, will do.

Reply:
Done